packages/drivers/odsp-driver/package.json

```diff
@@ -66,11 +66,11 @@
 		"@fluidframework/driver-base": "workspace:~",
 		"@fluidframework/driver-definitions": "workspace:~",
 		"@fluidframework/driver-utils": "workspace:~",
-		"@fluidframework/gitresources": "^1.0.1",
+		"@fluidframework/gitresources": "^2.0.0",
```

Comment:
Tangent: I don't see imports from gitresources in odsp-driver. @jatgarg do you know if it's actually required here?